application/actions/requests.go

```diff
@@ -27,195 +22,18 @@ func requestsList(c buffalo.Context) error {
-	output := make([]api.RequestAbridged, len(requests))
-
-	for i, request := range requests {
-		var err error
-		output[i], err = convertRequestToAPITypeAbridged(ctx, request)
-		if err != nil {
-			return []api.RequestAbridged{}, err
-		}
-	}
-
-	return output, nil
-}
-
-// converts model.Request into api.Request
-func convertRequestToAPIType(ctx context.Context, request models.Request) (api.Request, error) {
-	var output api.Request
-	if err := api.ConvertToOtherType(request, &output); err != nil {
-		err = errors.New("error converting request to api.request: " + err.Error())
-		return api.Request{}, err
-	}
-	output.ID = request.UUID
-
-	// Hydrate nested request fields for api.RequestAbridged
-	tx := models.Tx(ctx)
-	createdBy, err := hydrateRequestCreatedBy(ctx, request, tx)
-	if err != nil {
-		return api.Request{}, err
-	}
-	output.CreatedBy = &createdBy
-
-	origin, err := hydrateRequestOrigin(ctx, request, tx)
-	if err != nil {
-		return api.Request{}, err
-	}
-	output.Origin = &origin
-
-	destination, err := hydrateRequestDestination(ctx, request, tx)
-	if err != nil {
-		return api.Request{}, err
-	}
-	output.Destination = &destination
-
-	provider, err := hydrateProvider(ctx, request, tx)
-	if err != nil {
-		return api.Request{}, err
-	}
-	output.Provider = &provider
-
-	photo, err := hydrateRequestPhoto(ctx, request, tx)
-	if err != nil {
-		return api.Request{}, err
-	}
-	output.Photo = &photo
-
-	// TODO: hydrate other nested request fields after reconciling api.Request struct with the UI field list
-
-	return output, nil
-}
-
-// converts model.Request into api.RequestAbridged
-func convertRequestToAPITypeAbridged(ctx context.Context, request models.Request) (api.RequestAbridged, error) {
-	var output api.RequestAbridged
-	if err := api.ConvertToOtherType(request, &output); err != nil {
-		err = errors.New("error converting request to api.request: " + err.Error())
-		return api.RequestAbridged{}, err
-	}
-	output.ID = request.UUID
-
-	// Hydrate nested request fields
-	tx := models.Tx(ctx)
-	createdBy, err := hydrateRequestCreatedBy(ctx, request, tx)
-	if err != nil {
-		return api.RequestAbridged{}, err
-	}
-	output.CreatedBy = &createdBy
-
-	origin, err := hydrateRequestOrigin(ctx, request, tx)
-	if err != nil {
-		return api.RequestAbridged{}, err
-	}
-	output.Origin = &origin
-
-	destination, err := hydrateRequestDestination(ctx, request, tx)
-	if err != nil {
-		return api.RequestAbridged{}, err
-	}
-	output.Destination = &destination
-
-	provider, err := hydrateProvider(ctx, request, tx)
-	if err != nil {
-		return api.RequestAbridged{}, err
-	}
-	output.Provider = &provider
-
-	photo, err := hydrateRequestPhoto(ctx, request, tx)
-	if err != nil {
-		return api.RequestAbridged{}, err
-	}
-	output.Photo = &photo
-	
-	return output, nil
-}
+	// TODO evaluate assumption that user can only be logged in as a member of one organization
```

Comment:
While the statement is true (user can only be logged in as a member of one organization) that doesn't mean you can use only the first organization in the list. You'll need to iterate through all of them, I think.